### PR TITLE
Lavaland Overhaul: Part 1 - Frozen Darts

### PR DESCRIPTION
**Pull request by Super3222**
https://github.com/yogstation13/yogstation/pull/1773

#### Intent of your Pull Request

Watcher darts will freeze rock/asteroid walls upon hit. Nothing special, it just changes the color and description.

##### Changelog

:cl:
rscadd: Frozen Darts now leave a mark on rock walls in lavaland.
tweak: Frozen Darts freeze you again.
/:cl:

### CODE REVIEW

**Comment by matskuman5** [comment]
>Watcher darts will freeze asteroid walls upon hit.

>freeze asteroid walls

>asteroid

**Comment by Super3222** [comment]
Well they TECHNICALLY will.  It'll work for both lavaland and the asteroid.

**Comment by Keekenox** [comment]
Does it remove ebolamine from watcher darts? If not, get up on outta here.

**Comment by Time-Green** [comment]
This doesnt remove the bolamine.
REMOVE THE FUCKING BOLAMINE

**Comment by Keekenox** [comment]
Bolalmine is literally cancer

**Comment by Super3222** [comment]
Eh. Last time it was super nerfed to the point where watchers were a joke, and even further beyond that watchers froze you into a bag of arctic glacier ice blocks (on a LAVA based planet).

**Comment by ShadowDeath6** [comment]
just have it inject you with pyrosium

also the cryptobiolin is retarded remove that

**Comment by Keekenox** [comment]
Holy shot, no more chem watchers! There is absolutely nothing wrong with watchers as they are on TG. Leave it to yogstation to not only fix what isn't broken, but break it, tread on it, and hammer it into dust and snort it to fuel their mad addiction to nerfing and buffing random things until there isn't a shred of balance or logic in the whole game.

**Comment by Super3222** [comment]
@Keekenox It's pretty unfair to generalize everyone on here for a feature or two. Mac (a really old coder) decided an alternative to freezing on lavaland so he made a chemical that purely slowed you down. That practically nerfed watchers to hell. Adam came up with an alternative, and changed it to where the darts would also confuse you and make you stumble into the wrong direction. I haven't heard anyone complain, or to be honest with you even criticize it, up until now.

I could go through and just revert both of their changes so that watchers freeze again.

**Comment by Keekenox** [comment]
Randomly buffing and nerfing things is how you end up with how lavaland is now, yoga likes to fuck with existing content too much rather than adding content.

**Comment by Super3222** [comment]
I'm the exact opposite. I love adding content rather than buffing/nerfing, but one thing can't exist without the other.

**Comment by BOBsHERE** [comment]
Wait what? You haven't heard anyone complain about watchers and their cancer beams? That seems wrong.

**Comment by Super3222** [comment]
Yes. Most miners log off after dying and ragequitting because a watcher shot them in the groin 20 times making them fall back into a river of lava.

**Comment by Time-Green** [comment]
It takes one shot and you're movement is completely randomized

**Comment by ShadowDeath6** [comment]
how to fix: make epinephrine stabilize your temperature slowly

File: code/modules/projectiles/projectile/natural.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/obj/item/projectile/temp/basilisk
```
Comment (Cruix) [inline code/modules/projectiles/projectile/natural.dm:1]: Is this supposed to be obj/item/projectile/bullet/dart/basilisk?

**Comment by ShadowDeath6** [comment]
@Super3222 make epineph stabilize your temperature slowly

**Comment by Super3222** [comment]
Maybe in another PR or something. I've got to maintain a whole lot more, and this has been sitting here. I'm also pretty sure there's a chemical that's made for that

File: code/modules/projectiles/projectile/natural.dm (hunk 1 above)
Comment (Super3222) [reply code/modules/projectiles/projectile/natural.dm:1]: No, because it needs to be pathed as a temp projectile to lower theirr temperature.

**Pull request merged**